### PR TITLE
[zh-cn] Update services-networking/service.md

### PR DESCRIPTION
**Pull request by gaogao101**
Signed-off-by: Gao Qian <gaoq.jy@fujitsu.com>

### CODE REVIEW

File: content/zh-cn/docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -603,7 +603,7 @@ Note that the kube-proxy starts up in different modes, which are determined by i
 <!--
 ### User space proxy mode {#proxy-mode-userspace}
 
-In this mode, kube-proxy watches the Kubernetes control plane for the addition and
+In this (legacy) mode, kube-proxy watches the Kubernetes control plane for the addition and
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/services-networking/service.md:606]: The corresponding zh need to be changed.

File: content/zh-cn/docs/concepts/services-networking/service.md [hunk 2]
```diff
@@ -2386,13 +2411,13 @@ IPVS 专为负载均衡而设计，并基于内核内哈希表。
...
-about the API object at: [Service API object](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#service-v1-core).
+about the [Service API object](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#service-v1-core).
 
 ## Supported protocols {#protocol-support}
 -->
 ## API 对象   {#api-object}
 
-Service 是 Kubernetes REST API 中的顶级资源。你可以在以下位置找到有关 API 对象的更多详细信息：
+Service 是 Kubernetes REST API 中的顶级资源。你可以找到更多详细信息有关
 [Service 对象 API](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#service-v1-core).
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/services-networking/service.md:2421]:
```suggestion
Service 是 Kubernetes REST API 中的顶级资源。你可以找到有关
[Service 对象 API](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#service-v1-core)
的更多详细信息。
```

File: content/zh-cn/docs/concepts/services-networking/service.md [hunk 3]
```diff
@@ -1273,9 +1276,17 @@ IP 地址范围。
...
 （这也与早期的 Kubernetes 版本兼容。）
-请注意，此服务显示为 `<NodeIP>:spec.ports[*].nodePort` 和 `.spec.clusterIP:spec.ports[*].port`。
+
+{{< note >}}
+<!-- 
+This Service is visible as `<NodeIP>:spec.ports[*].nodePort` and `.spec.clusterIP:spec.ports[*].port`.
+If the `--nodeport-addresses` flag for kube-proxy or the equivalent field
+in the kube-proxy configuration file is set, `<NodeIP>` would be a filtered node IP address (or possibly IP addresses).
+-->
+此服务显示为 `<NodeIP>:spec.ports[*].nodePort` 和 `.spec.clusterIP:spec.ports[*].port`。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/services-networking/service.md:1286]:
```suggestion
此服务呈现为 `<NodeIP>:spec.ports[*].nodePort` 和 `.spec.clusterIP:spec.ports[*].port`。
```

**Comment by windsonsea** [comment]
Thanks.
/lgtm

**Pull request merged**